postgres/src/resources/application.conf

```diff
@@ -0,0 +1,306 @@
+# Copyright (c) 2014-2016 Snowplow Analytics Ltd. All rights reserved.
```

Comment:
do we want to keep the configuration in the application.conf file as the structure the snowplow follows is that they omit to add the conf file in application.conf and suggests to create our own conf, that will also be used in the docker images